consensus/state_sync/helpers.go

```diff
@@ -34,9 +34,9 @@ func (m *stateSync) sendToPeer(msg *anypb.Any, peerId cryptoPocket.Address) erro
 func getMessageType(msg *typesCons.StateSyncMessage) string {
 	switch msg.Message.(type) {
 	case *typesCons.StateSyncMessage_MetadataReq:
-		return "MetadataRequest"
```

Comment:
This is how it's being used:

```go
	fields := map[string]any{
		"height":     height,
		"peerId":     peerId,
		"proto_type": getMessageType(stateSyncMsg),
	}

	m.logger.Info().Fields(fields).Msg("Sending StateSync Message")
	return m.sendToPeer(anyMsg, peerId)
```

I think we should leave this as `MetadataRequest` OR rename `MetadataRequest` to `PeerMetadataRequest`, but `StateSyncMetadataResponse` seems wrong.